books/items.py

```diff
@@ -0,0 +1,12 @@
+# Define here the models for your scraped items
+#
+# See documentation in:
+# https://docs.scrapy.org/en/latest/topics/items.html
+
+import scrapy
+
+
+class BooksItem(scrapy.Item):
+    # define the fields for your item here like:
+    # name = scrapy.Field()
+    pass
```

Comment:
You should define fields here

Reply:
its automatically created files
Should i correct it?

Reply:
Yes. Look, comments give you a hint, that this module was created for the fields definition of your item.